### DISCUSSION TITLE
（已解决）缩略图突然无法加载

### DISCUSSION THREAD

**Discussion by 373578807** [AdNotFound/EhViewer#144] [Q&A]
![Screenshot_20241104_233452](https://github.com/user-attachments/assets/83ad9250-ee67-43ef-878b-7555dc907667)
看隔壁貌似是格式支持问题？ 点开后。直连情况可以下载和观看 但站内缩略图全挂了（不常用git 纯小白 不知道该如何正确反应问题 如果打扰到了还请见谅）
问题只出现在ex站点表站可以正常浏览缩略图

问题已解决  半夜看着dalao连夜更新了host   😭 感动 谢谢作者的付出

**Comment by Eiliot773** [comment]
哥们我也是这个问题，能告诉我解决方法吗？

**Reply by AdNotFound** [reply]
尝试更新到ci最新版
